recipe/field_mask_util.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git src/google/protobuf/util/field_mask_util.cc src/google/protobuf/util/field_mask_util.cc
+index 409010a..5c5e842 100644
+--- src/google/protobuf/util/field_mask_util.cc
++++ src/google/protobuf/util/field_mask_util.cc
+@@ -460,7 +460,7 @@ void FieldMaskTree::TrimMessage(const Node* node, Message* message) {
+       reflection->ClearField(message, field);
+     } else {
+       if (field->cpp_type() == FieldDescriptor::CPPTYPE_MESSAGE) {
+-        Node* child = node->children.at(field->name());
++        Node* child = node->children.find(field->name())->second;
```

Comment:
`std::map::at` was introduced in C++11. As VC2008 and VC2010 don't have C++11 support, a poor man's workaround is to use `std::map::find` to get an iterator and then get the value from that iterator.

**Note:** This is the only real way to do this as `std::map::operator[]` is **not** the same as `std::map::at` as it will **mutate** the `std::map` object by adding a new value if it is missing; whereas, `std::map::at` and `std::map::find` will not.